tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import shutil
+import subprocess
+from pathlib import Path, PurePath
+from typing import Optional, Protocol
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def project_root() -> PurePath:
```

Comment:
This was not technically needed since we run tox from the repo root in practice and I think tox enforces a CWD at tox.ini anyhow. It does always read a smagic to me though that relative paths like "pants" work. This just makes that path explicit. May not be worth it / may be just noise.

Reply:
I prefer the more explicit approach as most people (including me before reading this) don't know this is how Tox works.

Reply:
OK, great. Made perfectly explicit now by setting cwd to this file's parent.